docs/demo.md

```diff
@@ -0,0 +1,99 @@
+# Armada Demo
+
+This guide will show you how to take a quick test drive of an Armada
+instance already deployed to AWS EKS.
+
+## EKS
+
+The EKS demo of Armada can be found at these URLs:
+- [API Server](https://armada.demo.armadaproject.io)
```

Comment:
I put a comment in another branch but this link gives a 502 error.  I think its correct as we use it for the armadaUrl but the link isn't useful for a browser from my understanding.  We should remove.

Reply:
removed